src/genie/libs/parser/iosxr/show_ospfv3.py

```diff
@@ -544,13 +546,21 @@ def cli(self, output=None):
                 neighbor_rid = m.groupdict()['neighbor']
 
             # In the area 0 via interface GigabitEthernet0/0/0/0.1
+            # In the area 0 via interface GigabitEthernet0/0/0/1.500     BFD enabled, Mode: Default
             m = p3.match(line)
             if m:
                 area = int(m.groupdict()['area'])
                 interface = m.groupdict()['interface']
                 interface_dict = instance_dict.setdefault('area', {}).setdefault(area, {}). \
                     setdefault('neighbor_router_id', {}).setdefault(neighbor_rid, {}). \
                     setdefault('interface', {}).setdefault(interface, {})
+                
+                if m.groupdict()['enable'] and m.groupdict()['mode']:
```

Comment:
I am a little confused by these being grouped together. While the scenario of one existing without the other might not be possible, it would still be better to have these as two separate if-statements

Reply:
Hi @ThomasJRyan  - thanks for the review comments

I've split up the IF statements and added an 'or' in the schema